### ISSUE TITLE
Authelia integration

### ISSUE THREAD

**Issue by jzmini** [vicalloy/outline-docker-compose#36]
I really like this project. As an alternative to OIDC server, the authelia server would be nice.

**Comment by cr-sh** [comment]
yes please, switch the OIDC server to authelia, the current django implementation is messy: non admin users can't change their own passwords, sessions aren't invalidated after logout, and so on.
